wire-ios/Wire-iOS Tests/DeviceManagement/DeviceDetailsViewTests.swift

```diff
@@ -202,55 +221,90 @@ final class DeviceDetailsViewTests: ZMSnapshotTestCase, CoreDataFixtureTestHelpe
     }
 
     func testWhenE2eidentityViewIsDisabledInDarkMode() {
+        let mlsProvider = MockMLSProvider(isMLSEnbaled: false)
         let e2eIdentityProvider = MockE2eIdentityProvider()
         verify(matching:
                 setupWrappedInNavigationController(
                     mode: .dark,
                     e2eIdentityProvider: e2eIdentityProvider,
+                    mlsProvider: mlsProvider,
                     e2eIdentityCertificate: e2eIdentityProvider.certificate
                 )
         )
     }
 
     func testWhenE2eidentityViewIsEnabledAndCertificateIsValidInDarkMode() {
         let e2eIdentityProvider = MockValidE2eIdentityProvider()
+        let mlsProvider = MockMLSProvider(isMLSEnbaled: false)
         verify(matching:
                 setupWrappedInNavigationController(
                     mode: .dark,
                     e2eIdentityProvider: e2eIdentityProvider,
+                    mlsProvider: mlsProvider,
                     e2eIdentityCertificate: e2eIdentityProvider.certificate
                 )
         )
     }
 
     func testWhenE2eidentityViewIsEnabledAndCertificateIsRevokedInDarkMode() {
         let e2eIdentityProvider = MockRevokedE2eIdentityProvider()
+        let mlsProvider = MockMLSProvider(isMLSEnbaled: false)
         verify(
             matching: setupWrappedInNavigationController(
                 mode: .dark,
                 e2eIdentityProvider: e2eIdentityProvider,
+                mlsProvider: mlsProvider,
                 e2eIdentityCertificate: e2eIdentityProvider.certificate
             )
         )
     }
 
     func testWhenE2eidentityViewIsEnabledAndCertificateIsExpiredInDarkMode() {
         let e2eIdentityProvider = MockExpiredE2eIdentityProvider()
+        let mlsProvider = MockMLSProvider(isMLSEnbaled: false)
         verify(
             matching: setupWrappedInNavigationController(
                 mode: .dark,
                 e2eIdentityProvider: e2eIdentityProvider,
+                mlsProvider: mlsProvider,
                 e2eIdentityCertificate: e2eIdentityProvider.certificate
             )
         )
     }
 
     func testWhenE2eidentityViewIsEnabledAndCertificateIsNotActivatedInDarkMode() {
         let e2eIdentityProvider = MockNotActivatedE2eIdentityProvider()
+        let mlsProvider = MockMLSProvider(isMLSEnbaled: false)
+        verify(
+            matching: setupWrappedInNavigationController(
+                mode: .dark,
+                e2eIdentityProvider: e2eIdentityProvider,
+                mlsProvider: mlsProvider,
+                e2eIdentityCertificate: e2eIdentityProvider.certificate
```

Comment:
Why are you passing `e2eIdentityProvider` and `e2eIdentityCertificate` separately if the certificate is part of the `e2eIdentityProvider`?